enterprise/coderd/templates_test.go

```diff
@@ -922,6 +923,11 @@ func TestTemplateACL(t *testing.T) {
 
 	t.Run("everyoneGroup", func(t *testing.T) {
 		t.Parallel()
+
+		if !dbtestutil.WillUsePostgres() {
+			t.Skip("This test requires postgres")
```

Comment:
This smells to me like `dbmem` doesn't include the prebuilds user. That should be fixed; it's going to be maddening if `dbmem` and postgres are different in this respect.

Reply:
We've added support for the prebuilds system user to dbmem. A few tests began to fail. which is a good thing. We've addressed those.